freqtrade/freqai/freqai_interface.py

```diff
@@ -280,6 +305,12 @@ def start_backtesting(
                 dk.append_predictions(append_df)
             else:
                 if not self.model_exists(dk):
+                    if dk.backtest_live_models:
+                        raise OperationalException(
+                            "Training models is not allowed "
+                            "in backtest_live_models backtesting "
+                            "mode"
+                        )
```

Comment:
This will surely fit on fewer lines ...

that said - ain't we a bit deep into the process here to still operationalError? 
wouldn't half of the magic have already happened if we discover that the last pair of the pairlist doesn't have models available?